cmd/prometheus/main.go

```diff
@@ -305,6 +305,9 @@ func main() {
 	serverOnlyFlag(a, "storage.tsdb.wal-compression", "Compress the tsdb WAL.").
 		Hidden().Default("true").BoolVar(&cfg.tsdb.WALCompression)
 
+	serverOnlyFlag(a, "storage.tsdb.head-chunks-write-queue-size", "Size of the queue through which head chunks are written, 0 disables the queue completely. Experimental.").
```

Comment:
```suggestion
	serverOnlyFlag(a, "storage.tsdb.head-chunks-write-queue-size", "[EXPERIMENTAL] Size of the queue through which head chunks are written to the disk to be m-mapped, 0 disables the queue completely.").
```

Reply:
Perhaps this should be a feature flag instead of a separate flag? That way it is by default experimental and we are signalling that we hope it becomes the default in the future?

Reply:
just FYI I copied from here: https://github.com/prometheus/prometheus/blob/80724e5be47745a1af0072d00b176648e84c4a88/cmd/prometheus/main.go#L357
and
https://github.com/prometheus/prometheus/blob/80724e5be47745a1af0072d00b176648e84c4a88/cmd/prometheus/main.go#L360

If there is a better way to signal that something is experimental, maybe we should update these flags too

Reply:
I am fine with what is there now. Since it also controls the size I think just marking it as experimental similar to the other commands is reasonable.

Reply:
Can we add `to the disk to be m-mapped` part. (consider this to be a non-blocking comment)

Reply:
> Can we add to the disk to be m-mapped part. (consider this to be a non-blocking comment)

Done: c8b0a5609